src/redisai.c

```diff
@@ -1424,6 +1429,8 @@ int RedisModule_OnLoad(RedisModuleCtx *ctx, RedisModuleString **argv, int argc)
 
     RedisModule_SetModuleOptions(ctx, REDISMODULE_OPTIONS_HANDLE_IO_ERRORS);
 
+    RedisModule_SubscribeToServerEvent(ctx, RedisModuleEvent_Shutdown, RAI_CleanupModule);
```

Comment:
what is the minimal redis version supporting this event? does it align with our min_redis_version on our ramp files?

Reply:
yes, this API is supported from version 6.0.0, which is the minimal version for RedisAI anyway....